DAPDG v. 1.0.0

KonstantinGordeev opened the pull request:
После переделки алгоритма он снова не сходится, но буду еще менять

KonstantinGordeev committed:
v. 1.0.0

После переделки алгоритма он снова не сходится, но буду еще менять

alexrogozin12 reviewed (commented):
Сделай пожалуйста код в `.py` файле

KonstantinGordeev committed:
Add gapdg.py and example for class

GAPDG class based on BaseDecentralizedMethod and affine_constraints.ipynb with example of run

KonstantinGordeev committed:
Example run

KonstantinGordeev commented:
Переписал класс, теперь он наследуется от базового и использует оракла. Параметры подобрал перебором по неплотной сетке, попробую потом разобраться с правильной инициализацией. Алгоритм сходится через раз, но работает)
